### PR TITLE
WIP: Add awsosdmsp-quota-slice after registering a awsosdmsp clusterprofile.

### PR DESCRIPTION
**Pull request by samvarankashyap**
Should be merged after 
https://github.com/openshift/ci-tools/pull/2737 
Still work in progress.

### CODE REVIEW

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -901,7 +922,6 @@ secret_configs:
       item: insights-ci-account
     ops-mirror.pem:
       field: cert-key.pem
-      item: mirror.openshift.com
```
Comment (smg247) [inline core-services/ci-secret-bootstrap/_config.yaml:904]: did you mean to remove this?

Comment (samvarankashyap) [reply core-services/ci-secret-bootstrap/_config.yaml:904]: No, It was by accident. I will change it. @smg247

**Comment by smg247** [comment]
@samvarankashyap, now that github is working again you can check your test failures to show what is configured incorrectly. The `ci-secret-bootstrap-config-validation` test will make sure that secrets exist in vault for all of your new configs.

**Comment by samvarankashyap** [comment]
@smg247 Could you please take a look at the failed tests please. 
am not sure what am doing wrong. I see the items are in vault. 
may be am referencing it wrong.

**Comment by samvarankashyap** [comment]
Closing this pull request in favor of cluster profile.

**Pull request closed without merging**